vault/resource_transit_secret_backend_key.go

```diff
@@ -236,6 +263,61 @@ func transitSecretBackendKeyCreate(d *schema.ResourceData, meta interface{}) err
 	return transitSecretBackendKeyRead(d, meta)
 }
 
+func transitSecretBackendKeyImport(d *schema.ResourceData, meta interface{}) error {
+	client, e := provider.GetClient(d, meta)
+	if e != nil {
+		return e
+	}
+
+	backend := d.Get("backend").(string)
+	name := d.Get("name").(string)
+	autoRotatePeriod := getTransitAutoRotatePeriod(d)
+	data := map[string]interface{}{
```

Comment:
This data map is now duplicated in 3 places here,  `transitSecretBackendKeyCreate()`, and `transitSecretBackendKeyUpdate()`.  One suggestion would be to have `Create()`, call `Update()`, another would be to merge the two functions.